raiden-ts/tests/unit/raiden.spec.ts

```diff
@@ -82,6 +100,7 @@ function dummyReducer(state: RaidenState = dummyState) {
 
 function makeDummyDependencies(): RaidenEpicDeps {
   const provider = new JsonRpcProvider();
+  provider.getNetwork = jest.fn(async () => network);
```

Comment:
Nitpick, you can put `getNetwork` definition/mock inside the `Object.assign` below